spec/justifi/util_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+RSpec.describe Justifi::Util do
+  describe "#encode_parameters" do
+    context "should prepare parameters for an HTTP request" do
+      it do
```

Comment:
I do like following betterspecs but I'd say that it's put to a better use when testing resource related classes like [payments#spec](https://github.com/justifi-tech/justifi-ruby/blob/main/spec/justifi/payment_spec.rb)

Without `should`, the syntax would be like:
```ruby
  describe "#flatten_params" do
    context "encode parameters according to Rails convention" do
    ...
``` 
because I always try to leave the `describe -> context` .

For me, it feels more elegant to have all the class using this structure as it also feels like a protection policy ( basically trying to tell future coders that this test was scoped to a method within  `describe` and its behaviors would be handled on different contexts ), but that's just me. I'd be happy to establish a convention for our gem 😄